docs/understanding-airbyte/airbyte-protocol.md

```diff
@@ -1,29 +1,49 @@
 
 ## Key Concepts
+
 There are 2 major components in the Airbyte Protocol: Source and Destination. These components are referred to as Actors. A source is an application that is described by a series of standard interfaces. This application extracts data from an underlying data store. A data store in this context refers to the tool where the data is actually stored. A data store includes: databases, APIs, anything that produces data, etc. For example, the Postgres Source is a Source that pulls from Postgres (which is a data store). A Destination is an application that is described by a series of standard interfaces that loads data into a data store.
 
 The key primitives that the Protocol uses to describe data are Catalog, Configured Catalog, Stream, Configured Stream, and Field:
-* Stream - A Stream describes the schema of a resource and various metadata about how a user can interact with that resource. A resource in this context might refer to a database table, a resource in a REST API, or a data stream.
-* Field - A Field refers to a "column" in a Stream. In a database this would be a column; in a JSON object it is a field.
-* Catalog - A Catalog is a list of Streams that describes the data in the data store that a Source represents.
+
+- Stream - A Stream describes the schema of a resource and various metadata about how a user can interact with that resource. A resource in this context might refer to a database table, a resource in a REST API, or a data stream.
+- Field - A Field refers to a "column" in a Stream. In a database this would be a column; in a JSON object it is a field.
+- Catalog - A Catalog is a list of Streams that describes the data in the data store that a Source represents.
 
 An Actor can advertise information about itself with an [Actor Specification](#actor-specification). One of the main pieces of information the specification shares is what information is needed to configure an Actor.
 
 Each of these concepts is described in greater depth in their respective section.
 
+## Changelog
+
+The Airbyte Protocol is versioned independently of the Airbyte Platform, and the version number is used to determine the compatibility between connectors and the Airbyte Platform.
+
+| Version | Date       | Pull Request                                                                                                          | Subject                                                                          |
+| :------ | :--------- | :-------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------- |
+| 0.2.1   | 2022-09-09 | [16479](https://github.com/airbytehq/airbyte/pull/16479)                                                              | `AirbyteLogMessage.stack_trace` added                                            |
```

Comment:
This will be true once https://github.com/airbytehq/airbyte/pull/16479 is merged